lib/ffmpeg/decoderbase.py

```diff
@@ -91,7 +91,7 @@ def decode(self):
       )
 
     return call(
-      f"{exe2os('ffmpeg')} -v verbose {self.hwinit}"
+      f"{exe2os('ffmpeg')} -v verbose -strict -2 {self.hwinit}"
```

Comment:
The changes looks good to me. Since now native VVC decoder(native decoder then forks into VAAPI if it specificed) in ffmpeg is in EXPERIMENTAL status, which need this additional option in cmdline temporary. And once the EXPERIMENTAL flags removed in ffmpeg, I will submit a patch to fits to remove this additional option too.